src/routes/rooms.ts

```diff
@@ -49,10 +49,18 @@ router.post(
   asyncHandler(async (req, res) => {
     const body: CreateRoom = req.body;
```

Comment:
Still feels weird to have a type named with a verb.

Reply:
To be honest, I didn't tackle this on this PR, which name feels more natural to you?

Reply:
I don’t even know what this does (*cof* design doc *cof*) so my input won’t be too valuable

Reply:
jajajaja ok, fair enough, just FYI this is the handler of `POST /rooms` route, it's creating a room where the following object is expected:
```js
{
   "name":  "required",
   "description":  "optional I believe",
   "options": [ // This cannot be empty
      { "label": "", "value": "" }
   ]
}
```

Reply:
Lets' tackle this in other PR 🚀